gen/flights/loadouts.py

```diff
@@ -117,7 +117,11 @@ def iter_for(cls, flight: Flight) -> Iterator[Loadout]:
             pylons = payload["pylons"]
             yield Loadout(
                 name,
-                {p["num"]: Weapon.with_clsid(p["CLSID"]) for p in pylons.values()},
+                {
+                    p["num"]: Weapon.with_clsid(p["CLSID"])
+                    for p in pylons.values()
+                    if p["CLSID"] not in ("<CLEAN>", "")
+                },
```

Comment:
Why did this change? Doesn't this put the pylons back on clean loadouts?

Reply:
Thanks, I must have added the "<CLEAN>" special handling in loadouts.py by mistake, since it was there in weapons.py

I'll fix it.

Reply:
Doesn't this have the sample problem for `""` loadouts? If I've understood the bug correctly those are meaningful and must be preserved.

Reply:
When testing this, I didn't find any issues with the Mosquito loadouts in-game. I tested the following:
1. Rockets on wings, bombs in the internal bomb bay
2. Bombs on wings, bombs in the internal bomb bay
3. Mix of rockets and bombs on wings, bombs in the internal bomb bay

![image](https://user-images.githubusercontent.com/89945461/149655331-cae2b870-2fc3-4bb0-ac7e-3336cfa07b2b.png)

That's not to say there won't be other effects, but I wasn't able to find any problems with this.

Reply:
Did you configure those loadouts in Liberation, or in DCS as default loadouts? This code only affects the latter. If this isn't broken then I don't understand what this patch does.

Reply:
I have tested this with:
1. Starfire's Mosquito loadouts which include the "" weapons on wing hardpoints
2. Adding similar loadouts in my own custom loadouts in DCS (Saved Games\DCS\MissionEditor\UnitPayloads\MosquitoFBMkVI.lua)

Reply:
The above screenshot is from a Liberation campaign where I have tasked two Mosquito flights for CAS, with loadouts selected in the flight dialog. The first one has a mixed loadout with both bombs and rockets on wing pylons. The one in the background (left) has only rockets on wing pylons, leaving the #1 and #2 pylons with the clsid "" weapon.

Reply:
I'm going to have to play with this in the debugger to understand it then. This looks entirely broken to me.